middleware/integration_test/docker-compose.yml

```diff
@@ -0,0 +1,135 @@
+version: "3"
+services:
+  bitcoind:
+    image: nicolasdorier/docker-bitcoin:0.17.0
+    networks:
+      local:
+        ipv4_address: 10.10.0.10
+    environment:
+      EXPOSE_TCP: "true"
+      BITCOIN_EXTRA_ARGS: |
+        regtest=1
+        whitelist=0.0.0.0/0
+        server=1
+        regtest.rpcallowip=10.211.0.0/16
+        regtest.rpcallowip=172.17.0.0/16
+        regtest.rpcallowip=192.168.0.0/16
+        regtest.rpcallowip=0.0.0.0/16
+        regtest.rpcbind=0.0.0.0
+        rpcport=18443
+        rpcuser=rpcuser
+        rpcpassword=rpcpass
+    expose:
+      - "18443"
+      - "18444"
+    ports:
+      - "0.0.0.0:18443:18443"
+    volumes:
+      - "./volumes/bitcoin:/data"
+
+  electrs:
+    image: vulpemventures/electrs:latest
+    entrypoint:
+      - /build/electrs
+    command:
```

Comment:
I would move all the commands into an external shell script, then you can just do:
`command: sh mycommandscript.sh`

Reply:
I think I'll do this in a separate pull requset once we add some tooling to interact with it. Currently it should just run in the background, such that the middleware has something to connect to.

Reply:
It doesn't even need to be an interactive script, all it does is make the compose file a little cleaner :)